bitnami/mysql/README.md

```diff
@@ -56,10 +56,12 @@ It is strongly recommended to use immutable tags in a production environment. Th
 
 Bitnami will release a new chart updating its containers if a new version of the main container, significant changes, or critical vulnerabilities exist.
 
-### Use a different MySQL version
+### Use a different MySQL 8.x version
 
 To modify the application version used in this chart, specify a different version of the image using the `image.tag` parameter and/or a different repository using the `image.repository` parameter.
 
+> **Warning**: This chart is tested with MySQL version 8.x. Older versions most likely won't work if you merely switch the tag.
```

Comment:
```suggestion
### Use a different MySQL version

To modify the application version used in this chart, specify a different version of the image using the `image.tag` parameter and/or a different repository using the `image.repository` parameter.

> **Warning**: Each bitnami/mysql Helm chart is created using a specific MySQL version which is tested in our automated test & release pipeline. Substituting the version could cause degraded performance, broken chart features, or missing environment variables.  
```

I would opt for a more generic wording not only about 8.x

Reply:
@carrodher I think mentioning the specific version it is being tested on helps the user to realize it won't work on the version they are trying to use.